api-docs/Audit-Manager-Controller.yml

```diff
@@ -0,0 +1,170 @@
+---
+openapi: 3.0.1
+info:
+  title: Audit Manager Service
+  description: Rest Endpoints for operations related to audit
+  license:
+    name: Mosip
+    url: https://docs.mosip.io/platform/license
+  version: '1.0'
+servers:
+- url: https://api-internal.collab.mosip.net/v1/auditmanager
+  description: For Swagger
+tags:
+- name: Audit manager controller
+  description: Rest endpoints which consist of operations related to Audit Management
+paths:
+  "/audits":
+    post:
+      tags:
+      - Audit manager controller
+      summary: Persist a audit
+      description: persist a audit
+      operationId: addAudit
+      requestBody:
+        content:
+          application/json:
+            schema:
+              "$ref": "#/components/schemas/RequestWrapperAuditRequestDto"
+        required: true
+      responses:
+        '200':
+          description: Success or you may find errors in error array in response
+          content:
+            application/json:
+              schema:
+                "$ref": "#/components/schemas/ResponseWrapperAuditResponseDto"
+        '401':
+          description: Unauthorized
+        '403':
+          description: Forbidden
+        '404':
+          description: Not Found
+components:
+  schemas:
+    AuditRequestDto:
+      required:
+      - actionTimeStamp
+      - applicationId
+      - applicationName
+      - createdBy
+      - eventId
+      - eventName
+      - eventType
+      - hostIp
+      - hostName
+      - sessionUserId
+      type: object
+      properties:
+        eventId:
```

Comment:
```
eventId:
  type: string
  maxLength: 64
  minLength: 1
  description: Unique identifier for the event
```
Add descriptions to all schema properties to improve readability in Stoplight Docs